### PR TITLE
Adds UserGuide.md file 

### PR DESCRIPTION
**Pull request by kdunkerl**
Creates the UserGuide.md file explaining how our anonymize feature is used
Changes:
'UserGuide.md'
- creates file
- explains how to make a post anonymous
- explains how to turn an anonymous post back to non anonymous
- gives outline for testing explanation

Testing:
- not necessary since it's just the creation of a read me file

### CODE REVIEW

File: UserGuide.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature: Make a post anonymous
+How to use the feature:
+    Making a post anonymous:
+    - make a new post
```
Comment (rmdluo) [inline UserGuide.md]: should probably mention that it works on existing posts too (or at least I think it does)

File: UserGuide.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    - click the settings button to access the post tools
```
Comment (rmdluo) [inline UserGuide.md]: might be more accurate to say the post drop down cause I think there's a separate settings button

**Review by rmdluo — CHANGES_REQUESTED** [review]

File: UserGuide.md (hunk 1 above)
Comment (kdunkerl) [reply UserGuide.md]: changed wording in newest commit

File: UserGuide.md (hunk 2 above)
Comment (kdunkerl) [reply UserGuide.md]: changed wording in newest commit

**Pull request merged**